app/team.html

```diff
@@ -46,87 +46,104 @@ <h2 id="text_event_name"></h2>
-
-						  
-					   </span>
-				   </div>
-				   
-				   
+                  <div class="row">
+                      <strong>Preferred Team Size: </strong>                      
+                      <span>
+                         <span class="badge">
+                           {{param.currentTeamSize}}</span>
+                          
+                         <button><span class="glyphicon glyphicon-plus" ng-click="changeCurrentTeamSize(1);"></span></button>
+
+                          <button><span class="glyphicon glyphicon-minus" ng-click="changeCurrentTeamSize(-1);"></span></button>
+
+                          
+                       </span>
+                   </div>
+                   
+                   
 
                   </form>
 
 
-				<div class="row">
-                  <div class="col-sm-6">
-                     <h4>Team members</h4>
+        <div class="row">
+		
+			<div class="col-sm-3">
+				<h4>Team members</h4>
                      <!-- TODO: Show the current team member list -->
-					 
-					 <div ng-repeat="member in param.teamMembers">
-						<button type="button" class="btn" ng-click="removeMember(member);">
-							<span class="glyphicon glyphicon-remove" aria-hidden="true"></span> {{member}}
-						</button>
-					 </div>
-					 
-                  </div>
+                     
+				<div ng-repeat="member in param.teamMembers">
+					<button type="button" class="btn" ng-click="removeMember(member);">
+						<span class="glyphicon glyphicon-remove" aria-hidden="true"></span> {{member}}
+					</button>
+				</div>
+                     
+			</div>
 
-                  <div class="col-sm-6">
-                     <h4>Requests received:</h4>
-
-                     <!--TODO: Show the requests -->
-					 
-					 <div ng-repeat="r in requests">
-						<button type="button" class="btn" ng-click="processRequest(r);">
-							<span class="glyphicon glyphicon-backward" aria-hidden="true"></span> {{r}}
-						</button>
-					 </div>
-					 
-                  </div>
+			<div class="col-sm-3">
+				<h4>Requests received:</h4>
+				<div ng-repeat="r in requests">
+					<button type="button" class="btn" ng-click="processRequest(r);">
+						<span class="glyphicon glyphicon-backward" aria-hidden="true"></span> {{r}}
+					</button>
+				</div>
+			</div>
+		
+			<div class="col-sm-3">
+				<h4>Merge Requests received:</h4>
+				<div ng-repeat="q in mergeRequestReceived ">
+					<button type="button" class="btn" ng-click="processMergeRequest(q);">
+						<span class="glyphicon glyphicon-backward" aria-hidden="true"></span> {{q.$id}}
+					</button>
 				</div>
-				  
-				  
+			</div>
+
+			<div class="col-sm-3">
+				<h4>Your preferred team to merge</h4>
+				<div ng-repeat="t in param.teamToSelect " ng-show="t.$id != param.teamName">
+					<button type="button" class="btn" ng-click="teamMergeRequest(t);">
```

Comment:
I think there should be a button here for cancelling the merge request after sending the request.